picking_serial_list/models/stock_move_line.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, fields, api
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    forced_update_serial_qty = fields.Boolean()
+
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
ojo que es multi y lo estás tratando como un api one. tienes que pivotar sobre el self.
fijate en el @api de la funcion

Reply:
Yo creo que eso está bien. No tengo que pivotar sobre el self, sino sobre vals_list, que es una lista de diccionarios, independiente de self. El create original es este:
    @api.model_create_multi
    def create(self, vals_list):
        for vals in vals_list:
            if vals.get('move_id'):
                vals['company_id'] = self.env['stock.move'].browse(vals['move_id']).company_id.id
            elif vals.get('picking_id'):
                vals['company_id'] = self.env['stock.picking'].browse(vals['picking_id']).company_id.id

        mls = super().create(vals_list)

Reply:
totalmente de acuerdo 👍🏻